src/routes/index.js

```diff
@@ -1,14 +1,49 @@
-import { Auth } from "../modules/auth/pages";
-import { Main } from "../modules/main/pages";
+import { Auth } from "../modules/auth/pages/index";
```

Comment:
I would suggest doing an operation called in old days `barreling` and right now just declaring a single import entry point. In shortcut in all `modules` declare at the highest level an `index.js` file which only imports those elements, that should be imported outside the module and take them from this file and not from the file itself.